src/main/frontend/components/header.cljs

```diff
@@ -131,7 +132,8 @@
   [{:keys [on-click]}]
   (ui/with-shortcut :ui/toggle-left-sidebar "bottom"
     [:button.#left-menu.cp__header-left-menu.button.icon
-     {:on-click on-click}
+     {:title "Toggle left menu"
```

Comment:
Ideally any user-facing text should go through translation using `t` as logseq is available in 17 langs. I don't know if title attributes are too granular but I do see some of those translated in the codebase (thoughts @tiensonqin or @xyhp915 ?). If you decide to translate, you would put them in https://github.com/logseq/logseq/blob/master/src/main/frontend/dicts.cljc . The current namespacing scheme is a bit messy and some lack namespacing. Perhaps a namespace like `:header` or `:button`

Reply:
It'll be great if user-facing texts could have translations.